ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js

```diff
@@ -138,12 +138,10 @@ const mapStateToProps = (state, ownProps) => {
       });
   const isGasEstimate = getIsGasEstimatesFetched(state);
 
-  let customPriceIsSafe;
+  let customPriceIsSafe = true;
   if ((isMainnet || process.env.IN_TEST) && isGasEstimate) {
     customPriceIsSafe = isCustomPriceSafe(state);
-  } else if (isTestnet) {
-    customPriceIsSafe = true;
-  } else {
+  } else if (!(isMainnet || process.env.IN_TEST || isTestnet)) {
```

Comment:
Don't we need to check if gasEstimates have been fetched yet in this block as well?

Reply:
Thanks. I added this check in the most recent commit.